### ISSUE TITLE
React Keys: Replace uuid() with crypto.randomUUID()

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28699]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Our lesson introduces the uuid package but there's no reason to import that whole package in that use case when you can use the built-in `crypto.randomUUID()` for the same thing instead.

## Acceptance criteria

- [ ] Code examples have any `uuid()` mentions replaced with `crypto.randomUUID()` (you cannot destructure `randomUUID` unfortunately).
- [ ] Any lesson text mentioning `uuid()` are changed to reference `crypto.randomUUID()`.
- [ ] The sentence linking to the uuid package is changed to refer and link to [crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-react-new-keys-in-react

**Comment by Asartea** [comment]
Would be willing to take this one

**Comment by MaoShizhong** [comment]
Go for it
